resurfemg/postprocessing/quality_assessment.py

```diff
@@ -193,3 +194,72 @@ def percentage_under_baseline(
     valid_timeproducts = percentages_aub < aub_threshold
 
     return valid_timeproducts, percentages_aub
+
+
+def consecutive_manoeuvres(
+        signal,
+        start_s,
+        end_s,
+        fs,
+        manoeuvres_idxs
+):
+    """
+    Detect manoeuvres (for example Pocc) with no supported breaths
+    in between. First identify the supported breaths from the ventilator signal
+    and then check for in-between supported breaths.
+    If no supported breaths are detected in between two manoeuvres,
+    a 'double dip' is detected.
+    Double_dips true means the manoeuvre is not valid.
+
+    :param signal: signal in which the manoeuvre is performed
+    :type signal: ~numpy.ndarray
+    :param start_s: start second of the window in which to be searched
+    :type start_s: ~int
+    :param end_s: end second of the windown in which to be searched
+    :type end_s: ~int
+    :param fs: sampling frequency
+    :type fs: ~int
+    :param manoeuvres_idxs : list of manoeuvres indexes
+    :type manoeuvres_idxs: ~list
+
+    :returns: valid_manoeuvres, double_dips
+    :return type: list, list
+    """
+
+    start_s_samp = fs * start_s
+    end_s_samp = fs * end_s
+    V_t = signal[int(start_s_samp):int(end_s_samp)]
+    treshold = 0.25 * np.percentile(V_t, 90)
+    prominence = 0.10 * np.percentile(V_t, 90)
+    width = 0.25 * fs
+    resp_eff, _ = scipy.signal.find_peaks(V_t, height=treshold,
+                                          prominence=prominence, width=width)
+
+    treshold = 0.5 * np.percentile(V_t[resp_eff], 90)
+    prominence = 0.5 * np.percentile(V_t, 90)
+    width = 0.25 * fs
+    resp_eff2, _ = scipy.signal.find_peaks(
+        V_t, height=treshold, prominence=prominence, width=width)
+
```

Comment:
I would split this function in two:
1. the upper part in event_detecton.detect_supported_breaths(...)
2. And the bottom part in quality_assessment.consecutive_manoeuvre(peaks_s_maneoeuvre, peaks_s_regular_breathing):